cli/kata-check_ppc64le.go

```diff
@@ -75,3 +93,14 @@ func archKernelParamHandler(onVMM bool, fields logrus.Fields, msg string) bool {
 func getCPUDetails() (vendor, model string, err error) {
 	return genericGetCPUDetails()
 }
+
+func isSMTOff() bool {
+	// Check if the SMT is off
+	out, err := exec.Command(ppc64CpuCmd, smtStatusOption).Output()
+
+	if err == nil && strings.TrimRight(string(out), "\n") == "SMT is off" {
```

Comment:
Is `ppc64_cpu` i18n'd? Do we need to set `LC_ALL=C` / `LANG=C` to ensure this output will match in English?

Reply:
No it's not i18n'd.

Reply:
Ok - thanks for the responses @mpe. I wonder if it would be a good precaution to set the C locale anyway "just in case".

Let's get some more reviews on this PR: ping @kata-containers/runtime.

Reply:
Yeah it's probably sensible to use `LANG=C` just in case.